recipe/meta.yaml

```diff
@@ -25,14 +24,17 @@ requirements:
     - python  >=3.6
     - black  >=20.0.0,<21.0.0
     - toml
-    - typing-extensions  # [py<38]
-    - dataclasses        # [py<37]
+    - typing-extensions
+    - dataclasses
+  run_constrained:
+    - isort  >=5.0.1
```

Comment:
@akaihola I am applying a constraint on the isort dependency, this would be picked up only if users install isort.